server/app/files.go

```diff
@@ -350,6 +351,10 @@ func (a *App) CopyAndUpdateCardFiles(boardID, userID string, blocks []*model.Blo
 	newFileNames, err := a.CopyCardFiles(boardID, blocks, asTemplate)
 	if err != nil {
 		a.logger.Error("Could not copy files while duplicating board", mlog.String("BoardID", boardID), mlog.Err(err))
+		return err
```

Comment:
This seems to be change of logic. Previously no error would cause the function to return, but now it does. Why is this needed?

Reply:
Yes, nice catch. Will update it.